src/sdam/description/topology/server_selection/test/in_window.rs

```diff
@@ -0,0 +1,194 @@
+use std::{collections::HashMap, sync::Arc, time::Duration};
+
+use approx::abs_diff_eq;
+use semver::VersionReq;
+use serde::Deserialize;
+use tokio::sync::RwLockWriteGuard;
+
+use crate::{
+    options::StreamAddress,
+    runtime::AsyncJoinHandle,
+    sdam::{description::topology::server_selection, Server},
+    selection_criteria::ReadPreference,
+    test::{
+        run_spec_test,
+        EventClient,
+        FailCommandOptions,
+        FailPoint,
+        FailPointMode,
+        TestClient,
+        CLIENT_OPTIONS,
+        LOCK,
+    },
+    RUNTIME,
+};
+
+use super::TestTopologyDescription;
+
+#[derive(Debug, Deserialize)]
```

Comment:
Just to make sure, these tests aren't covered by the unified runner, right?

Reply:
nope, this is a new format